main.go

```diff
@@ -37,6 +38,7 @@ func mainError() error {
 	var err error
 
 	var f Flag
+	flag.BoolVar(&f.EniConfigureRouting, "eni-configure-routing", false, "If set to true, app will use also setup routing for eni to prevent asymetric routing.")
```

Comment:
Why not doing this always?

Reply:
You mean why the default is false or why we even allow such a setting?

It's a good question, I cannot easily imagine situation when we would don't want to do it.